maquetado/tech.scss

```diff
@@ -25,10 +25,17 @@
   display: flex;
   flex-direction: column;
   margin: 10px 4px;
+  min-height: 300px;
+  padding: 20px;
   position: relative;
   text-align: center;
   width: 200px;
 
+  &:hover::before {
+    left: 10px;
```

Comment:
Isn't a good practice to do transitions with this properties because they modify the `layout` the navigator. I recommend you this [article](https://developers.google.com/web/fundamentals/performance/rendering/)